src/Yoti/Util/ExtraData/ThirdPartyAttributeConverter.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Yoti\Util\ExtraData;
+
+use Yoti\Entity\AttributeDefinition;
+use Yoti\Entity\AttributeIssuanceDetails;
+use Yoti\Exception\ExtraDataException;
+use Yoti\Sharepubapi\IssuingAttributes;
+use Yoti\Sharepubapi\ThirdPartyAttribute as ThirdPartyAttributeProto;
+
+class ThirdPartyAttributeConverter
+{
+    /**
+     * @param string $value
+     *
+     * @return \Yoti\Entity\AttributeIssuanceDetails
+     */
+    public static function convertValue($value)
+    {
+        $thirdPartyAttributeProto = new ThirdPartyAttributeProto();
+        $thirdPartyAttributeProto->mergeFromString($value);
+
+        $token = $thirdPartyAttributeProto->getIssuanceToken();
+        if (empty($token)) {
+            throw new ExtraDataException('Failed to retrieve token from ThirdPartyAttribute');
+        }
+
+        $expiryDate = null;
+        $issuingAttributes = [];
+
+        $issuingAttributesProto = $thirdPartyAttributeProto->getIssuingAttributes();
+
+        if ($issuingAttributesProto instanceof IssuingAttributes) {
+            $parsedDateTime = \DateTime::createFromFormat(
+                AttributeIssuanceDetails::DATE_FORMAT_RFC3339,
+                $issuingAttributesProto->getExpiryDate(),
```

Comment:
Is it worth moving `DATE_FORMAT_RFC3339` into the global constants?  Or is this version of the standard only used by `AttributeIssuanceDetails`?

Reply:
Fixed in 80e9400